diff --git a/src/tcompiler.cpp b/src/tcompiler.cpp
index 9ef0c26..f8a40de 100644
--- a/src/tcompiler.cpp
+++ b/src/tcompiler.cpp
@@ -1265,13 +1265,13 @@ struct FunctionEmitter {
                             printf("%s%s",s.c_str(), (fstate == f) ? "\n" : " ");
                         }
                     } while(fstate != f);
-                    CU->mi->run(scc.begin(), scc.end());
+                    // CU->mi->run(scc.begin(), scc.end()); // inliner
                     for(size_t i = 0; i < scc.size(); i++) {
                         VERBOSE_ONLY(T) {
                             std::string s = scc[i]->getName();
                             printf("optimizing %s\n",s.c_str());
                         }
-                        CU->fpm->run(*scc[i]);
+                        // CU->fpm->run(*scc[i]); // per-function optimizer
                         VERBOSE_ONLY(T) {
                             scc[i]->dump();
                         }
